doc/classes/Particles2D.xml

```diff
@@ -39,14 +40,17 @@
 			How rapidly particles in an emission cycle are emitted. If greater than [code]0[/code], there will be a gap in emissions before the next cycle begins. Default value: [code]0[/code].
 		</member>
 		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps">
+			This setting can be used to set the particle system to render at a fixed FPS. For instance, changing the value to [code]2[/code] will make the particles render at 2 frames per second. Note this does not slow down the particle system itself.
 		</member>
 		<member name="fract_delta" type="bool" setter="set_fractional_delta" getter="get_fractional_delta">
+			This can be used to turn Fract Delta on or off.
```

Comment:
I'd remove the "This can be used to" from the beginning, personally. It doesn't sound right when I'm reading this in the context of documentation, to have "This can be used to" as part of the description, unless preceding an example.